perun-base/src/test/resources/test-schema.sql

```diff
@@ -88,16 +88,16 @@ create table cabinet_publications (
 	id integer not null,
 	externalId integer not null, --identifier at externa pub. system
 	publicationSystemId integer not null, --identifier of external pub. system (cabinet_publication_systems.id) *)
-	title varchar(1024) not null,
+	title longvarchar not null,
 	year integer not null, --short title of publication
-	main varchar(4000), --full cite of publication
-	isbn varchar(32),
+	main longvarchar, --full cite of publication
+	isbn longvarchar,
 	categoryId integer not null, --identifier of category (cabinet_categories.id)
-	createdBy varchar(1300) default user not null,
+	createdBy longvarchar default user not null,
 	createdDate timestamp not null,
 	rank numeric (38,1) default 0 not null,
-	doi varchar(256),
-	locked varchar(1) default 0 not null,
+	doi longvarchar,
+	locked longvarchar default 0 not null,
```

Comment:
Should be probably char(1) and default to '0' as in postgres.

Reply:
fixed